runhouse/servers/cluster_servlet.py

```
@@ -81,6 +81,8 @@ async def __init__(
         # been created yet, since it is a freshly initialized cluster servlet. In this case, we will return an empty string.
         self._is_log_file_ready = False
 
+        self._are_multinode_servlets_ready = False
```

whats the signfiicance of this variable if its not a multinode cluster? (we also have the `is_multinode` variable)

---

Since the creation on the node_servlets are async, there might be a case when the head node trying to collect the status from those nodes before they are ready, and then we are gonna get a `Failed to look up actor` exception. If this is the first round of cluster status checks, this exception is acceptable, and it should not be logged / cause any breakage. This variable helps us in deciding whether to ignore the exception or not.

`is_multinode` is just a flag that helps parsing the status data, and a nice-to-know cluster information.  (If we would like to know whether the cluster is multi-node we could just get the `is_multinode` value, instead of checking the amount of cluster IPs.